test/quantization/test_quant_api.py

```diff
@@ -401,43 +336,6 @@ def test_gptq_quantizer_int4_weight_only(self):
             f"accuracy regressed from 7.76 to {result['results']['wikitext']['word_perplexity,none']}"
         )
 
-    @unittest.skip("skipping until we get checkpoints for gpt-fast")
-    def test_quantizer_int4_weight_only(self):
```

Comment:
Does this PR just move the location of these classes, if so this test should still pass right, or am I missing something? @danielpatrickhug @HDCharles Is there a reason why we want to remove this test?

Reply:
We don't want 2 int4 implementations